tutorials/source-modeling/plot_eeg_mri_coords.py

```diff
@@ -95,7 +95,7 @@
 #         >>> pos_vox = ...  # loaded from a file somehow
 #         >>> img = nibabel.load(fname_T1)
 #         >>> vox2mri = img.header.get_vox2ras_tkr()  # voxel -> mri
-#         >>> vox2mri[:3] /= 1000.  # mm -> m
+#         >>> vox2mri[:3, -1] /= 1000.  # translation column: mm -> m
```

Comment:
I'm actually not so sure this is really correct, because probably the scaling should not only be applied to the _translation_, but also to the … well, to the "scaling part" of the transformation matrix. That is, the diagonal of `vox2mri[:3, :3]` should be multiplied with `1/1000`… But I haven't really worked with affine transformations "hand-on" in may years so I'm a little rusty :)

Reply:
It's going to depend on whether or not the affine changes the data units I think. In this case it does (voxel to physical), so this now looks incorrect